pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     "openai < 2",
     "oracledb < 3",
     "philter-lite < 1",
-    "pyarrow < 19",
+    "pyarrow < 20",
```

Comment:
Unrelated, I just stole this from a [dependabot PR](https://github.com/smart-on-fhir/cumulus-etl/pull/375) that was consistently failing CI for no good reason.